README.md

```diff
@@ -106,15 +106,44 @@ It's important to note that if a miner's proof of storage is not valid, it indic
 
 
 # 5. Default Configuration
-- **CHUNK_SIZE**: Default value is `1<<22`. This represents the size of a chunk in bytes. The data is partitioned into chunks of this size (4MB) for storage and retrieval.
-- **DEFAULT_N_CHUNKS**: Default value is `1<<8`. This represents the minimum number of chunks a miner should provide at least. (1GB)
-- **THRESHOLD**: Default value is `0.9`. This represents the maximum amount of space the miner can use based on available space. It's used to determine the size of the partition for data storage.
-- **db_root_path**: Default value is `'~/bittensor-db'`. This is the path where the SQLite databases for data storage and retrieval are stored.
-- **netuid**: Default value is `7`. This is the netuid of the storage subnet that the miner and validator are serving on.
+## 5.1 Allocator (allocate.py)
+- **db_root_path**: Default value is `'~/tensorage-db'`. This is the path where the SQLite databases for data storage and retrieval are stored.
+- **size_in_gb**: Default value is `100GB`. This is the default size to store data.
+- **validator**: Default value is `False`. If `True`, only generate hash DB for validators.
+- **disable_prompt**: Default value is `False`. If `True`, not wait for user input to confirm the allocation.
```

Comment:
The "no_prompt" parameter has only been renamed to "disable_prompt" to avoid conflicts with the "no_prompt" parameter of wallets as of Bittensor version 6.8.0.